protos/protos/public/modeldb/versioning/VersioningService.proto

```diff
@@ -468,6 +468,18 @@ message FindRepositoriesBlobs {
     }
 }
 
+message GetUrlForVersionedBlob {
+    RepositoryIdentification repository_id = 1;
+    string commit_sha = 2;
+    repeated string location = 3;
+    string path_dataset_component_blob_path = 4;
+    string method = 5; //This is either GET or PUT
+
```

Comment:
This needs `uint64 part_number` [for multipart upload](https://github.com/VertaAI/modeldb/blob/f3540fc7378f8690e3331275a690c4fa90e42a38/protos/protos/public/modeldb/CommonService.proto#L111) (see Slack thread)